integrations/getUserProfile/index.test.ts

```diff
@@ -0,0 +1,40 @@
+import fetch from "node-fetch";
+
+import { get as getEnvValue } from "../../lib/env";
+
+describe("getUserProfile", () => {
+  const backendBaseurl = getEnvValue("IO_BACKEND_HOST");
+  const endpoint = `${backendBaseurl}/profile`;
+
+  test("is unauthorized without sessionToken", async () => {
+    const headers = {};
+    const expectedHttpCode = 401;
+    const expectedBody = "Unauthorized";
+
+    const client = () => fetch(endpoint, { headers });
```

Comment:
these calls should use the generated types (using io-utils) + the relative client 
in the case of the app backend you can mutuate it from 

https://github.com/pagopa/io-app/blob/master/ts/api/backend.ts

(I agree this should be generated automatically and put in a package : )

Reply:
I didn't for a couple of reasons:

1. `io-utils`'s type system enforce me to not call an endpoint in a way it shouldn't (example: cannot call an authenticated endpoint without authentication). This is good at application level, but is not here because it doesn't allow me to stress the service with unexpected calls.
2. I'd prefer to leave the test suite as low-level as I can. In the end, the subject of tests is a restful service, thus I want each test to express clearly what kind of http call is going to perform.

Reply:
That's a very good point ! While I agree that it's ok to use your own client (to force invalid requests), in case of successful calls that would be useful to check that the response payloads aligns to the type the client is expecting from the server (and the client is using the generated types). This is useful even to check that there are no API breaking changes deployed. What about a compromise ? (use your own client for calls, check the response payload against generated types for successful calls)

Reply:
@gunzip yes, that's actually what I meant to do (but I'm having some type-troubles thou 😓).
So:
1. I perform _good ol'_ http request
2. I check `response` object's http code
3. I check if the response body (if any) respects the structure defined by the spec - using generated types and `io-ts` decoders.

How does is sound? Am I missing something?

Reply:
sounds right !

Reply:
I resolve the comment as I implemented this is `getServerInfo` suite